### PR TITLE
Refactor master items component

### PR DESCRIPTION
**Pull request by kprakobkit**

### CODE REVIEW

**Comment by achiu8** [comment]
Sorry to nit, but can we name the spec files with pascal case so they match the components (`ComponentName_spec` vs. `Component_Name_spec`)?

File: src/components/MasterItems.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  addComment(e) {
+    const comment = e.target.value;
+
+    this.setState({ comment });
+  }
+
+  render() {
+    return dom.div(
+      null,
+      dom.h1(null, 'New Order'),
```
Comment (achiu8) [inline src/components/MasterItems.js]: Should this still be part of the `NewOrder` component? Or at least use a `prop` so we can reuse for adding entries to an existing order, like you mentioned?

File: src/components/NewOrder.js [hunk 2]
```diff
@@ -55,34 +37,11 @@ class NewOrder extends Component {
...
-        )
-      ),
-      dom.p(
-        null,
-        dom.button(
-          { className: 'btn btn-default add-entry btn-lg btn-block', onClick: this.addEntry },
-          'Add Item'
-        )
-      ),
+      MasterItems(_.extend({}, {
```
Comment (achiu8) [inline src/components/NewOrder.js]: Why do we need to `extend` here?

Comment (kprakobkit) [inline src/components/NewOrder.js]: Good catch. No reason. Oversight on my part. Same thing for file name. I'll make the change.

File: src/components/MasterItems.js (hunk 1 above)
Comment (kprakobkit) [inline src/components/MasterItems.js]: You mean like pass the title as a prop to the component? If so, I agree.

**Pull request merged**